membershipextras.php

```diff
@@ -369,9 +369,8 @@ function membershipextras_civicrm_pageRun($page) {
 function membershipextras_civicrm_validateForm($formName, &$fields, &$files, &$form, &$errors) {
   $formAction = $form->getAction();
   $isNewMembershipForm = ($formName === 'CRM_Member_Form_Membership' && ($formAction & CRM_Core_Action::ADD));
-  $isRenewMembershipForm = ($formName === 'CRM_Member_Form_MembershipRenewal' && ($formAction & CRM_Core_Action::RENEW));
   $isPaymentPlanPaymentlanPayment = CRM_MembershipExtras_Utils_InstalmentSchedule::isPaymentPlanWithSchedule();
-  if (($isNewMembershipForm || $isRenewMembershipForm) && $isPaymentPlanPaymentlanPayment) {
+  if ($isNewMembershipForm && $isPaymentPlanPaymentlanPayment) {
```

Comment:
can we change this variable $isPaymentPlanPaymentlanPayment name to $isPaymentPlanPayment